soh/soh/GameMenuBar.cpp

```diff
@@ -201,12 +194,110 @@ namespace GameMenuBar {
+                        ImGui::Text("FPS: Original (20)");
+                    } else {
+                        ImGui::Text("FPS: %d", currentFps);
+                        if (currentFps == 30) {
+                            fpsSlider = 2;
+                        } else { // currentFps == 60
+                            fpsSlider = 3;
+                        }
+                    }
+                    if (CVarGetInteger("gMatchRefreshRate", 0)) {
+                        UIWidgets::DisableComponent(ImGui::GetStyle().Alpha * 0.5f);
+                    }
+
+                    if (ImGui::Button(" - ##WiiUFPS")) {
+                        fpsSlider--;
+                    }
+                    ImGui::SameLine();
+                    ImGui::SetCursorPosX(ImGui::GetCursorPosX() - 7.0f);
+
+                    UIWidgets::Spacer(0);
+
+                    ImGui::SliderInt("##WiiUFPSSlider", &fpsSlider, 1, 3, "", ImGuiSliderFlags_AlwaysClamp);
+
+                    ImGui::SameLine();
+                    ImGui::SetCursorPosX(ImGui::GetCursorPosX() - 7.0f);
+                    if (ImGui::Button(" + ##WiiUFPS")) {
+                        fpsSlider++;
+                    }
+
+                    if (CVarGetInteger("gMatchRefreshRate", 0)) {
+                        UIWidgets::ReEnableComponent("");
+                    }
+                    if (fpsSlider > 3) {
+                        fpsSlider = 3;
+                    } else if (fpsSlider < 1) {
+                        fpsSlider = 1;
+                    }
+
+                    if (fpsSlider == 1) {
+                        currentFps = 20;
+                    } else if (fpsSlider == 2) {
+                        currentFps = 30;
+                    } else if (fpsSlider == 3) {
+                        currentFps = 60;
+                    }
+                    CVarSetInteger("gInterpolationFPS", currentFps);
+                    SohImGui::RequestCvarSaveOnNextTick();
+                #else
+                    bool matchingRefreshRate =
+                        CVarGetInteger("gMatchRefreshRate", 0) && SohImGui::WindowBackend() != SohImGui::Backend::DX11;
+                    UIWidgets::PaddedEnhancementSliderInt(
+                        (currentFps == 20) ? "FPS: Original (20)" : "FPS: %d",
+                        "##FPSInterpolation", "gInterpolationFPS", minFps, maxFps, "", 20, true, true, false, matchingRefreshRate);
+                #endif
+                    UIWidgets::Tooltip(
+                        "Uses Matrix Interpolation to create extra frames, resulting in smoother graphics. This is purely "
+                        "visual and does not impact game logic, execution of glitches etc.\n\n"
+                        "A higher target FPS than your monitor's refresh rate will waste resources, and might give a worse result.\n\n"
+                        "For consistent input lag, set this value and your monitor's refresh rate to a multiple of 20."
```

Comment:
is this true? i didn't know/notice

Reply:
What part do you mean? If you mean the consistent input lag, I've taken that from the old tooltip because I also wasn't sure about how true it was.

Reply:
Yeah, I meant the consistent input lag. I tried to do comments per-line but gh isnt making that super clear

I figure even if it is true it's something that belongs in more long form docs than a tooltip

Reply:
I've just removed it. If it's actually a thing, I can't imagine it being very impactful as I've also never noticed it when playing with 144 FPS.